index.js

```diff
@@ -20,18 +36,38 @@ var prefs = Cc["@mozilla.org/preferences-service;1"].getService(Ci.nsIPrefServic
 var Request = require("sdk/request").Request;
 var tabs = require("sdk/tabs");
 
+const DEBUG = true;
+const TAG = "bitcoin-price-ticker";
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).